sapi/fpm/tests/response.inc

```diff
@@ -91,6 +91,37 @@ class Response
         return $this;
     }
 
+    /**
+     * Expect that one of the processes in json status process list has a field with value that
+     * matches the supplied pattern.
+     *
+     * @param string $fieldName
+     * @param string $pattern
+     *
+     * @return Response
+     */
+    public function expectJsonBodyPatternForStatusProcessField(string $fieldName, string $pattern)
+    {
+        $rawData = $this->getBody('application/json');
+        $data = json_decode($rawData, true);
+        if (empty($data['processes']) || !is_array($data['processes'])) {
+            $this->error(
+                "The body data is not a valid status json containing processes field '$rawData'"
+            );
+        }
+        foreach ($data['processes'] as $process) {
+            if (preg_match('/' . $pattern . '/', $process[$fieldName]) !== false) {
```

Comment:
I recommend using `()` as the delimiter so that the caller doesn't need to escape `/` or any other literal character.

Reply:
Is it for compatibility with Windows?

Reply:
Ah yeah will use pipes in case it was used for matching the whole uri...

Reply:
I'm actually using pipes elsewhere - not sure why I used slashes here...